scripts/labs/argocd/kind-argo.sh

```diff
@@ -258,6 +258,11 @@ spec:
       values: |
         serviceMonitor:
           enabled: true
+        initContainer:
+          extraArgs:
+            - --loggingFormat=json
+        extraArgs:
+          - --loggingFormat=json
```

Comment:
why 2 extraArgs ? one is for main container ?

Reply:
yes, init container and kyverno container